pkg/webhook/server/server.go

```diff
@@ -282,16 +284,29 @@ func (s *Server) mutate(ctx context.Context, obj runtime.Object) (runtime.Object
 		return nil, errors.New("request is not of type apiextensions v1")
 	}
 	review.Response = s.MutationWebhook.Mutate(ctx, review.Request)
+	s.logAdmissionReview(review, "mutate")
+
 	return review, nil
 }
 
+func (s *Server) logAdmissionReview(review *admissionv1.AdmissionReview, operation string) {
+	if review.Request == nil {
+		s.log.Info(operation + " request handled nil request")
```

Comment:
I think this should be logged at debug rather than info level.
This is not really an expected situation, so maybe log something like `unexpected nil request`